ansible_ai_connect/main/settings/base.py

```diff
@@ -622,10 +622,7 @@ def is_ssl_enabled(value: str) -> bool:
 # Playbook Generation/Explanation endpoints
 # ------------------------------------------
 ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT = (
-    os.getenv(
-        "ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT", "False" if DEPLOYMENT_MODE == "on-prem" else "True"
-    ).lower()
-    == "true"
+    True
```

Comment:
we don't want to hardcode this

Reply:
we still want it to be able to take env setting

Reply:
@hasys yea, updated the same